src/js/components/Page/Page.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useMemo } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Box } from '../Box';
+import { PageContext } from './PageContext';
+import { ResponsiveContext } from '../../contexts/ResponsiveContext';
+import { PagePropTypes } from './propTypes';
+
+const Page = ({ kind, ...rest }) => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  const value = useMemo(
+    () => ({
+      alignSelf: theme.page[kind]?.alignSelf,
+      fill: 'horizontal',
+      width: theme.page[kind]?.width,
+      pad: theme.page[kind]?.pad?.[size],
+      margin: theme.page[kind]?.margin?.[size],
+    }),
+    [theme, size, kind],
+  );
+
+  return (
+    <PageContext.Provider value={value}>
+      <Box
+        width={
+          theme.page[kind].width?.min
```

Comment:
Can we just pass `theme.page[kind].width`? It will handle whether it's a string or an object.

Reply:
I ended up removing the width from this box, realized it wasn't needed here